### ISSUE TITLE
NameError: name 'CUDA_RUNTIME_LIB' is not defined

### ISSUE THREAD

**Issue by Teoge** [TimDettmers/bitsandbytes#338]
```
===================================BUG REPORT===================================
Welcome to bitsandbytes. For bug reports, please run

python -m bitsandbytes

 and submit this information together with your error trace to: https://github.com/TimDettmers/bitsandbytes/issues
================================================================================
Traceback (most recent call last):
  File "/opt/conda/envs/pt12/lib/python3.9/runpy.py", line 188, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/opt/conda/envs/pt12/lib/python3.9/runpy.py", line 147, in _get_module_details
    return _get_module_details(pkg_main_name, error)
  File "/opt/conda/envs/pt12/lib/python3.9/runpy.py", line 111, in _get_module_details
    __import__(pkg_name)
  File "/opt/conda/envs/pt12/lib/python3.9/site-packages/bitsandbytes/__init__.py", line 7, in <module>
    from .autograd._functions import (
  File "/opt/conda/envs/pt12/lib/python3.9/site-packages/bitsandbytes/autograd/__init__.py", line 1, in <module>
    from ._functions import undo_layout, get_inverse_transform_indices
  File "/opt/conda/envs/pt12/lib/python3.9/site-packages/bitsandbytes/autograd/_functions.py", line 9, in <module>
    import bitsandbytes.functional as F
  File "/opt/conda/envs/pt12/lib/python3.9/site-packages/bitsandbytes/functional.py", line 17, in <module>
    from .cextension import COMPILED_WITH_CUDA, lib
  File "/opt/conda/envs/pt12/lib/python3.9/site-packages/bitsandbytes/cextension.py", line 13, in <module>
    setup.run_cuda_setup()
  File "/opt/conda/envs/pt12/lib/python3.9/site-packages/bitsandbytes/cuda_setup/main.py", line 101, in run_cuda_setup
    binary_name, cudart_path, cuda, cc, cuda_version_string = evaluate_cuda_setup()
  File "/opt/conda/envs/pt12/lib/python3.9/site-packages/bitsandbytes/cuda_setup/main.py", line 382, in evaluate_cuda_setup
    cudart_path = determine_cuda_runtime_lib_path()
  File "/opt/conda/envs/pt12/lib/python3.9/site-packages/bitsandbytes/cuda_setup/main.py", line 242, in determine_cuda_runtime_lib_path
    warn_in_case_of_duplicates(conda_cuda_libs)
  File "/opt/conda/envs/pt12/lib/python3.9/site-packages/bitsandbytes/cuda_setup/main.py", line 215, in warn_in_case_of_duplicates
    f"Found duplicate {CUDA_RUNTIME_LIB} files: {results_paths}.. "
NameError: name 'CUDA_RUNTIME_LIB' is not defined
```

CUDA 11.3, python 3.9, pytorch 1.12.1, bitsandbytes 0.38.0
But bitsandbytes 0.37.0 works fine

**Comment by Teoge** [comment]
Oh, it's fixed, I didn't notice

**Comment by wuzeyuuu** [comment]
Buddy, how you fixed it ?

**Comment by Teoge** [comment]
@wuzeyuuu The bug was fixed in https://github.com/TimDettmers/bitsandbytes/commit/659a7dfc7165b166b7972250c39daa0b90ad501d, but somehow it wasn't included in the 0.38.0 pip release. You may roll back to 0.37.0 or build the package with the latest source to solve this issue.
